opensearch_py_ml/ml_commons/ml_common_client.py

```diff
@@ -5,6 +5,9 @@
 # Any modifications Copyright OpenSearch Contributors. See
 # GitHub history for details.
 
+
```

Comment:
Not for this line.
The file name seems mismatch with ml-commons plugin name https://github.com/opensearch-project/ml-commons

How about change to `ml_commons_client` ?